index.html

```diff
@@ -0,0 +1,23 @@
+
+
+<h1 id = "title"> 
+    PEDRO JOAQUIN CHAMORRO CARDENAL
+    </h1>
+  <br>
+  
+  <Main id = "main">
+  <link rel="stylesheet" href="styles.css">
+      <div id= "img-div" div> 
+        <img id = "image"> <img src ="https://www.confidencial.digital/wp-content/uploads/2022/01/PEDRO-JOAQUIN-CHAMORRO1.jpg.webp"> <figcaption id ="img-caption" >Last picture taken to Pedro Joaquin Chamorro </figcaption>
+  <head> 
+    <div id="tribute-info">
+    <ul>
+   <li>   Pedro Joaquin Chamorro exemplified the values and principles of full democracy and free elections: political pluralism and accountability; no successive terms; separation of public from private interests; the struggle against corruption; reforms with social justice; freedom of the press.</li>
+    
```

Comment:
Please have in mind that a ul and li tag are used to make a list. A list consist of several items in it. Here we just have one item in the list (so technically is not a list). If you want to make a list, please add more items to the list, otherwise you can put this text inside a paragraph tag